packages/odie-client/src/utils/zendesk-message-converter.ts

```diff
@@ -1,5 +1,5 @@
 import { __ } from '@wordpress/i18n';
-import { Message, MessageRole, MessageType, ZendeskMessage } from '../types';
+import { Context, Message, MessageRole, MessageType, ZendeskMessage } from '../types';
```

Comment:
Can we add the directive `type` here?

```suggestion
import type { Context, Message, MessageRole, MessageType, ZendeskMessage } from '../types';
```

Both perform the same operation, but the typing definition is not included in the compiled version, it's just used while coding 🙂